tests/utils.py

```diff
@@ -591,6 +591,8 @@ def run(self, result=None):
             '-5',
             '-3',
             '5',
+            '1',
+            '-1'
```

Comment:
I added 1 to verify that the Int.__eq__(Bool) ran properly. Adding those showed me that there was an error in the implementation for __eq__ which I've now modified. 
I didn't have a specific reason for adding the -1. I'll remove that.